README.md

```diff
@@ -97,7 +97,7 @@ The following developer guides cover specific topics for a technical audience.
 - [Meeting Events](https://aws.github.io/amazon-chime-sdk-js/modules/meetingevents.html)
 - [Integrating Amazon Voice Focus and Echo Reduction Into Your Application](https://aws.github.io/amazon-chime-sdk-js/modules/amazonvoice_focus.html)
 - [Adding Frame-By-Frame Processing to an Outgoing Video Stream](https://aws.github.io/amazon-chime-sdk-js/modules/videoprocessor.html)
-- [Adding Background Filtering to an Outgoing Video Stream](https://aws.github.io/amazon-chime-sdk-js/modules/backgroundfilter_video_processor.html)
+- [Adding Background Filtering to an Outgoing Video Stream](https://aws.github.io/amazon-chime-sdk-js/modules/backgroundfilter_videofx_processor.html)
```

Comment:
The NPM's page will have an incorrect link if you update the link in README.md after the release. Can you keep the same link?

Reply:
What do you mean? this is just point to the new link which only available after the release right?

Reply:
Discuss offline. This is fine to keep!